client/src/css/Aaron.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+h1,
+p,
+Table {
+  font-family: 'Inter', sans-serif;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h1 {
+  padding-top: 15vh;
+  font-size: 60;
+}
+
+body {
+  background-color: #daad86;
+}
```

Comment:
don't worry about this affecting other pages as of right now. danielle and I are working on a way to separate the styling of different pages, will update you when we have that figured out